.circleci/config.yml

```diff
@@ -48,8 +48,9 @@ references:
         name: Install Neo4j
         command: |
           curl -fsSL https://debian.neo4j.com/neotechnology.gpg.key | sudo gpg --dearmor -o /usr/share/keyrings/neo4j.gpg
-          echo "deb [signed-by=/usr/share/keyrings/neo4j.gpg] https://debian.neo4j.com stable latest" | sudo tee -a /etc/apt/sources.list.d/neo4j.list
+          echo "deb [signed-by=/usr/share/keyrings/neo4j.gpg] https://debian.neo4j.com stable 4.4" | sudo tee -a /etc/apt/sources.list.d/neo4j.list
```

Comment:
need to make sure it's synced with whatever we have in `pyproject.toml`, so we don't update one and forget about the other